src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const arrSort = [];
```

Comment:
```suggestion
const sortedArray = [];
```
можна назвати наприклад так, або по іншому, але без зайвих скорочень